docs/_attic/WIP-lamborghini-distribution/transactions.md

```diff
@@ -76,7 +76,7 @@ When a validator wishes to withdraw their transaction fees it must send
 triggered each with any change in individual delegations, such as an unbond,
 redelegation, or delegation of additional tokens to a specific validator. This
 transaction withdraws the validators commission rewards, as well as any rewards
-earning on their self-delegation. 
+earning on their self-delegation.
```

Comment:
... we can just delete this whole directory, we're not going to implement the Lamborghini model, unless there's a reason to keep it around. Certainly we don't need to update it, since it doesn't reflect any implementation.

Reply:
I think that's why it's in "attic". Should we just wipe this @rigelrozanski ?

Reply:
I think it's a good idea to wipe... @ValarDragon wanted it in here for reference a while ago... either way I've added it to a new repo https://github.com/rigelrozanski/lamborghini-distribution

Reply:
Sounds like the best of both worlds to me! We should delete it here then.